instrumentation/all/opentelemetry-instrumentation-all.gemspec

```diff
@@ -27,6 +27,7 @@ Gem::Specification.new do |spec|
   spec.required_ruby_version = '>= 3.0'
 
   spec.add_dependency 'opentelemetry-instrumentation-active_model_serializers', '~> 0.21.0'
+  spec.add_dependency 'opentelemetry-instrumentation-active_storage', '~> 0.0.0'
```

Comment:
I think it's best to include this instrumentation to -all and -rails once this pr gets merged and released

Reply:
I think that's a good call. However, if this is added to -rails, I don't think it also has to be added to -all, because -all requires -rails.

Reply:
Make sense. In https://github.com/open-telemetry/opentelemetry-ruby-contrib/pull/1313/commits/d6979a7eff054d8e7f2c67dd62e0929eff4b3554 , I removed -active_storage from -all .
I'll create another PR to -active_storage this to -rails after this PR is relesed.